core/test/XiosRead_test.cpp

```diff
@@ -107,7 +107,7 @@ MPI_TEST_CASE("TestXiosRead", 2)
     // Create HField instances to read the data into
     HField field_2D(ModelArray::Type::H);
     field_2D.resize();
-    HField field_3D(ModelArray::Type::Z);
+    ZField field_3D(ModelArray::Type::Z);
```

Comment:
Technically, this is purely a textual error. In this context, `ZField` and `HField` both resolve to `ModelArray`. Also, don't forget that ZField is being removed entirely Real Soon Now™.